### ISSUE TITLE
Current state of the project

### ISSUE THREAD

**Issue by qur2** [edge/fluxette#15]
Hi,

I'm looking at your library an it really looks nice. I'm still a bit puzzled by the "why" section of the readme (https://github.com/edge/fluxette#why) because there is no mention of the `redux` project, and obviously both are very similar.
What are the differences between `fluxette` and `redux`?
So far, I saw the following:
- the `reducer` dependency has different types of reducers 
- if I understood correctly, it is possible to dispatch multiple actions at once
- dispatching returns a handle on the action processing and it is possible to know when they are done without looking into the state

Thanks for your feedback.
Cheers,
Aurélien.

**Comment by qur2** [comment]
Also, I found out it's kind of written for React 0.13 (`props.children()` as a function call and various deprecated methods). Is there any plan to upgrade for use with 0.14 or 15?

Thanks,
Aurélien.

**Comment by alkhe** [comment]
Hey @qur2, glad to see that this project is still getting some hits.

I saw Dan's talk back in June, and in July I decided to create a Flux implementation with a deep foundation in functional concepts. Redux was simply an inspiration, and there were big differences between the two libraries at first. Even though I experimented independent of any knowledge of Redux's development, the two projects converged on what is probably the true Functional Flux architecture, which is basically a _recursive reduce + emit change_ pattern. We came up with different names for the same concepts (e.g. Leaf => end reducer, Shape => combineReducers, select => reselect, Context => Provider), and produced strikingly similar APIs. In fact, most of the Redux ecosystem was fully compatible by pure coincidence. I gave a talk about it in September, where you can gain a little context about the whole situation: http://rdc-ff.surge.sh/.

Eventually fluxette just became a simpler and less verbose version of Redux (I remember @taion calling it a "super concise redux" back when I first posted in the Reactiflux slack), but at this point I decided that eventually the Redux people would figure everything out and reach where I wanted fluxette to go, so I dropped the project.

I hope I answered some of your questions, I had to rush this answer to meet an appointment.

**Comment by qur2** [comment]
Hi,

Thanks for the explanation. I was attracted to the project because of the
name (I speak French :) and I found the documentation very clear and
concise. Also, the different types of reducers made it more clear to me
than combineReducers. But I then I played around with redux few months ago,
when there was the transition between react 13 and 14 and I think it
evolved quite a bit since then, specifically the documentation.
So I guess I'll go with redux then, I should be able to understand it
better now.

Thank you,
Aurélien.

**Comment by qur2** [comment]
As far as I'm concerned, this ticket can be closed. I'm leaving that to you: I thought you might want to leave it open for better visibility. That could help people wondering about the project.

Cheers,
Aurélien.

**Comment by alkhe** [comment]
Yeah, I'll probably leave it open for other people who might stumble across this project.
